packages/react/src/emotion-element.js

```diff
@@ -11,6 +11,26 @@ import { serializeStyles } from '@emotion/serialize'
 const sanitizeIdentifier = (identifier: string) =>
   identifier.replace(/\$/g, '-')
 
+export const getLabelFromStackTrace = (stackTrace: string): ?string => {
+  if (!stackTrace) return undefined
+
+  // console.log(stackTrace)
+
+  // Chrome — TODO: why is it like this?
```

Comment:
`React.forwardRef` creates an object with a `render` property that is later on called by React. If we mimic this with:
```js
onst test = { render: function MyComp() { console.log(new Error().stack) } }
test.render()
```
we'll receive such a stack trace:
```
Error
    at Object.MyComp [as render] (<anonymous>:1:56)
    at <anonymous>:2:6
```

This was fixed in this PR: https://github.com/emotion-js/emotion/pull/1973

As to the `(?:Object\.|Module\.|)` part. If we use this:
```js
const test = { MyComp() { console.log(new Error().stack) } }
test.MyComp()
```
then we end up with:
```
Error
    at Object.MyComp (<anonymous>:1:39)
    at <anonymous>:2:6
```

There is also a chance that a component gets exported from an actual module and this is when the Module part kicks in. The Module part was originally added in response to this issue: https://github.com/emotion-js/emotion/issues/1947

We can also find such a test case from the old issues: https://github.com/emotion-js/emotion/pull/1714#discussion_r365518850